addons/l10n_in_edi/views/account_move_views.xml

```diff
@@ -10,6 +10,13 @@
                 <field name="l10n_in_edi_cancel_reason" invisible="country_code != 'IN' or state != 'posted' or not l10n_in_edi_show_cancel"/>
                 <field name="l10n_in_edi_cancel_remarks" invisible="country_code != 'IN' or state != 'posted' or not l10n_in_edi_show_cancel"/>
             </xpath>
+             <xpath expr="//header" position="after">
+                <field name="display_null_pan_warning" invisible="1"/>
+                <div class="alert alert-warning main mb-0" role="alert"
+                        invisible="not display_null_pan_warning">
+                        If the partner has no PAN Number, please apply TDS / TCS at a higher rate.
```

Comment:
To put a constraint, we have to set a higher rate for each TDS/TCS section. But since there's no place on the tax form for the system to check these rates, we can't put constraint.

Also, there are exceptions, like payments to non-residents who might not have PAN. In these cases, charging TDS/TCS at a higher rate isn't necessary even without PAN. So, if we make a rule, it might stop the process. Instead, we're just issuing a warning without making any firm rules.
![2024-05-01_10-35](https://github.com/odoo/odoo/assets/152843893/33107f76-d6ad-4030-baaf-50d59b7de7f9)